app/client/src/pages/AdminSettings/Profile/UserProfileImagePicker.tsx

```
@@ -0,0 +1,86 @@
+import type Uppy from "@uppy/core";
+import { ReduxActionErrorTypes } from "ee/constants/ReduxActionConstants";
+import type { ErrorActionPayload } from "sagas/ErrorSagas";
+import { USER_DISPLAY_PICTURE_FILE_INVALID } from "ee/constants/messages";
+
+function FormDisplayImage() {
+  // TODO: Fix this the next time the file is edited
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const [file, setFile] = useState<any>();
+  const dispatch = useDispatch();
+  const user = useSelector(getCurrentUser);
+  const dispatchActionRef = useRef<(uppy: Uppy.Uppy) => void | null>();
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)